### PR TITLE
77 add new functions to estimationspy

### PR DESCRIPTION
**Pull request by lmantill**

### CODE REVIEW

**Comment by lmantill** [comment]
@vatai Please check this short PR when you have time :)

File: src/radicalpy/estimations.py [hunk 1]
```diff
@@ -460,6 +421,85 @@ def k_ST_mixing(Bhalf: float) -> float:
     return -C.g_e * (C.mu_B * 1e-3) * Bhalf / C.h
 
 
+def k_electron_transfer(R: float, deltaG: float = -1, lambd: float = 1) -> float:
+    """Electron transfer rate.
+
+    The default values return the maximum electron transfer rate (lambd = -deltaG).
```
Comment (vatai) [inline src/radicalpy/estimations.py:427]: That `lambd = - deltaG` is a bit confusing? Do you mean to say they have the opposite sign? Then you should write something like `lambd * deltaG < 0` or `sgn(lambda) = - sgn(deltaG)`. And why is the `a` missing from lambda (at the end)?

File: src/radicalpy/estimations.py [hunk 2]
```diff
@@ -460,6 +421,85 @@ def k_ST_mixing(Bhalf: float) -> float:
...
+    return 0.5 * (-b + np.sqrt(b**2 + 48 * k_escape**2 * MFE))
+
+
+def k_reencounter(encounter_dist: float, diff_coeff: float) -> float:
+    """Re-encounter rate.
+
+    Source: `Salikhov, J. Magn. Reson., 63, 271-279 (1985)`_.
+
+    Args:
+            encounter_dist (float): The effective re-encounter distance (R*) (m).
```
Comment (vatai) [inline src/radicalpy/estimations.py:491]: Is this `R*` the symbol/name used in the literature? This is what we talked about? right? To use a more verbose names for programming, and mention the 1-2 letter symbols in the docs, right?

**Review by vatai — APPROVED** [review]
@lmantill Reviewed! I have 3 comments:
- a suggestion: default value for eta in `diffusion_coefficient`
- 2 questions about `lambd` in `k_electron_transfer` and
- a question about the `R*` in the `encounter_dist` parameters docstring in `k_reencounter`?  Is an example an example of what we talked about, to have more verbose variable/argument names in the code, and mention the symbol used in physics/chemistry only in the docstrings?

**Pull request merged**